components/datetime/src/pattern/names.rs

```diff
@@ -1997,22 +1997,6 @@ impl<FSet: DateTimeNamesMarker> RawDateTimeNames<FSet> {
         Ok(())
     }
 
-    fn load_mz_periods<P>(
```

Comment:
Nit: I prefer the previous style of separate functions for loading separate keys. One important decision here is that we use `Default` as the DataRequest, and I don't want to inline that assumption to a bunch of places.

Just keep this function and make it return the DataResponseMetadata